MAINTAINERS.md

```diff
@@ -0,0 +1,10 @@
+## Maintainers
+
+| Maintainer | GitHub ID | Affiliation |
+| --------------- | --------- | ----------- |
+| Barani Bikshandi | [bbarani](https://github.com/bbarani) | Amazon |
+| Peter Zhu | [peterzhuamazon](https://github.com/peterzhuamazon) | Amazon |
+| Peter Nied | [peternied](https://github.com/peternied) | Amazon |
+| Sayali Gaikawad | [gaiksaya](https://github.com/gaiksaya) | Amazon |
```

Comment:
Am I able to get added to this list? I'll get more notifications and such this way.

Reply:
Hi @DandyDeveloper,

We are currently getting a decision out for adding external maintainers. Please stay tune for more updates. If we eventually add you there will be another PR to add your name here.

In the meantime please feel free to read the maintainer file for your responsibility:
https://github.com/opensearch-project/.github/blob/main/MAINTAINERS.md

Thanks.